src/app/(post-login)/submission/program/[shortName]/clinical-data/ClinicalEntityDataTable.tsx

```diff
@@ -636,71 +638,159 @@ const ClinicalEntityDataTable = ({
 			borderBottom: `1px solid ${theme.colors.grey_2}`,
 		};
 
+		const headerStyle = css`
+			background-color: ${theme.colors.grey_4};
+			border-bottom: 1px solid ${theme.colors.grey_2};
+			border-right: 1px solid ${theme.colors.grey_2};
+			padding: 5px;
+			font-size: 13px;
+			text-align: left;
+		`;
+
+		const stickyCSS = css`
+			position: sticky !important;
+			left: 0;
+			top: 0;
+			z-index: 1;
+			background-color: white;
+		`;
+
 		columns = [
 			{
 				id: 'clinical_core_completion_header',
-				header: (
-					<div
-						css={css`
-							display: flex;
-							align-items: center;
-							justify-content: center;
-							position: relative;
-						`}
-					>
-						CLINICAL CORE COMPLETION
-						<Tooltip
-							style={{ position: 'absolute', left: 'calc(100% - 20px)', top: '-2px' }}
-							html={
-								<p
-									css={css`
-										margin: 0px;
-										margin-right: 6px;
-									`}
-								>
-									For clinical completeness, each donor requires: <br />
-									DO: at least one Donor record <br />
-									PD: at least one Primary Diagnosis record <br />
-									NS: all the registered Normal DNA Specimen record <br />
-									TS: all the registered Tumour DNA Specimen record <br />
-									TR: at least one Treatment record <br />
-									FO: at least one Follow Up record <br />
-								</p>
-							}
+				meta: { customHeader: true },
+				sortingFn: sortEntityData,
+				header: (props) => {
+					return (
+						<th
+							colSpan={props.colSpan}
+							css={css`
+								${headerStyle};
+								border-right: 3px solid ${theme.colors.grey};
+							`}
 						>
-							<Icon name="question_circle" fill="primary_2" width="18px" height="18px" />
-						</Tooltip>
-					</div>
-				),
+							<div
+								css={css`
+									display: flex;
+									align-items: center;
+									justify-content: center;
+									position: relative;
+								`}
+							>
+								CLINICAL CORE COMPLETION
+								<Tooltip
+									style={{ position: 'absolute', left: 'calc(100% - 20px)', top: '-2px' }}
+									html={
+										<p
+											css={css`
+												margin: 0px;
+												margin-right: 6px;
+											`}
+										>
+											For clinical completeness, each donor requires: <br />
+											DO: at least one Donor record <br />
+											PD: at least one Primary Diagnosis record <br />
+											NS: all the registered Normal DNA Specimen record <br />
+											TS: all the registered Tumour DNA Specimen record <br />
+											TR: at least one Treatment record <br />
+											FO: at least one Follow Up record <br />
```

Comment:
not a request for change, because I know this is an established pattern and not something just from this PR:
🤔 this dictionary right here is what I wish we do with VS' intellisense, that if you hover over a constant, it also gives you a description of what it means, apart from its string value.
wonder if there's a plugin to do that sort of thing.

Reply:
ahh JSDoc to the rescue! no plugin needed 👍 
<img width="339" alt="image" src="https://github.com/icgc-argo/rdpc-ui/assets/2107110/1e9e36d6-e525-414f-8e8c-eeae304cfb14">

Reply:
nice!